client/src/App.js

```diff
@@ -1,29 +1,60 @@
-import { useEffect, useState } from 'react';
-import logo from './logo.svg';
 import './App.css';
+import Navbar from './components/Navbar';
+import Home from './pages/Home';
+import SearchResult from './pages/SearchResult';
+import Favorites from './pages/Favorites';
+import backgroundImage from '../src/pages/serge-le-strat-rS4OSc9yhSo-unsplash.jpg';
+import { Route, Routes } from 'react-router-dom';
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 
 function App() {
-  const [serverData, setServerData] = useState("");
+  // console.log(window.location);
+  const [search, setSearch] = useState('');
+  const [recipes, setRecipes] = useState([]);
+  const APP_ID = process.env.REACT_APP_ID;
```

Comment:
I would usually pull constants like this out of a function and put them at the top level, usually right after the imports.